### DISCUSSION TITLE
Line chart - dots showing instead of lines. Set connectgap defaults to true

### DISCUSSION THREAD

**Discussion by HuifangYeo** [atoti/atoti#622] [General]
## Description

I'm plotting a line chart on datetime (up to second level) split by members. The lines becomes dots.
![image](https://user-images.githubusercontent.com/35210495/91372438-3a330b80-e846-11ea-98f0-039a37d25ef1.png)

However, if I filter the members, it's a connected line.
![image](https://user-images.githubusercontent.com/35210495/91372497-5c2c8e00-e846-11ea-91b3-5b88f9e43e70.png)

I set [connectgaps](https://plotly.com/python/line-charts/#connect-data-gaps) to true in the plotly configuration:
```
 "plotly": {
		"data": {
			"commonTraceOverride": {
				"connectgaps": true
			}
		}
	}
```
The lines are now joined as expected:
![image](https://user-images.githubusercontent.com/35210495/91372657-c9d8ba00-e846-11ea-99ba-2add2c6804b5.png)

Would it be better to set connectgaps to true?

**Comment by mtadams007** [comment]
Hi @HuifangYeo,

Could you share a notebook with this issue? I'm unable to reproduce it on my side.

Thanks!

**Comment by HuifangYeo** [comment]
Hi @mtadams007,
 
I'll sorry I've overlooked this issue. I don't have the use case that uses the date anymore.
However, below is a similar one. You can look at the last chart in the notebook.

![image](https://user-images.githubusercontent.com/35210495/125910802-b8d2ecb7-8596-4ef7-956a-3fa7ab95b3e4.png)


[wine-modeling.zip](https://github.com/atoti/atoti/files/6828740/wine-modeling.zip)

**Comment by bloussou** [comment]
Issue is still here in `0.6.4`
